testsuite/model/pom.xml

```diff
@@ -102,6 +102,12 @@
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-core</artifactId>
+            <type>test-jar</type>
+            <version>${infinispan.version}</version>
+        </dependency>
```

Comment:
Please add this to the dependency management in the root `pom.xml` together with the other Infinispan dependencies. I'd like to have those in one central place.